bin/dm_parse_ea.py

```diff
@@ -48,24 +49,74 @@ def read_in_logfile(path):
     return log_file
 
 
+def clean_logfile(log_file):
+    # Remove the button press registered during scanner responses
+    scan_response = ["101", "104"]
+    indexes_to_drop = []
+
+    for index, row in log_file.iterrows():
+        if ("rating" in log_file["Code"][index]) and any(
+            resp in log_file["Code"][index - 2] for resp in scan_response
+        ):
+            indexes_to_drop.append(index)
+            indexes_to_drop.append(index - 2)
+
+    if len(indexes_to_drop) == 0:
+        log_file_cleaned = log_file
+    else:
+        log_file_cleaned = log_file.drop(log_file.index[indexes_to_drop])
+        log_file_cleaned = log_file_cleaned.reset_index(drop=True)
+        logger.warning(
+            "Removed registered rating occurred before or after actual rating"
+        )
```

Comment:
Maybe include the number of "chunks" being dropped here for QC purposes (i.e if an abnormally large number of indices got dropped). 

```suggestion
        logger.warning(
            f"Removed {len(indexes_to_drop)/2} registered ratings that occurred before or after actual rating"
        )
```